dotCMS/src/main/java/com/dotcms/util/pagination/TemplateView.java

```diff
@@ -0,0 +1,410 @@
+package com.dotcms.util.pagination;
+
+import java.util.Date;
+
+public class TemplateView {
```

Comment:
Why not extend it from Template and add the missing properties.

Reply:
Because the template is a webasset and retrieve so much